patterns/spirv.hexpat

```diff
@@ -0,0 +1,33 @@
+#include "spvopcodes.hexpat"
+#include "spvgenerators.hexpat"
+
+struct Version {
+    u8 dummy1;
```

Comment:
Instead of dummy variables you can also use the `padding[N]` syntax to skip N bytes forwards in the parsing

Reply:
Ah cool, yeah thats a bit nicer! updated :)